treatment-service/config/config.go

```diff
@@ -88,10 +88,28 @@ type Monitoring struct {
 	MetricLabels []string       `json:"metric_labels" default:""`
 }
 
+// MessageQueueKind describes the message queue for transmitting event updates to and fro Treatment Service
+type MessageQueueKind = string
```

Comment:
Since this is appearing in multiple places, shall we move these definitions to the `common` module, under a `messagequeue` package?